projects/aca-content/src/lib/components/info-drawer/metadata-tab/metadata-tab.component.ts

```diff
@@ -39,27 +39,36 @@ import { Actions, ofType } from '@ngrx/effects';
   imports: [CommonModule, ContentMetadataModule],
   selector: 'app-metadata-tab',
   template: `
-    <adf-content-metadata-card
+    <adf-content-metadata
       [readOnly]="!canUpdateNode"
       [preset]="'custom'"
       [node]="node"
       [displayAspect]="displayAspect$ | async"
+      [displayTags]="true"
+      [displayCategories]="true"
       [(editable)]="editable"
+      [(editableTags)]="editableTags"
+      [(editableCategories)]="editableCategories"
+      [(group)]="group"
     >
-    </adf-content-metadata-card>
+    </adf-content-metadata>
   `,
   encapsulation: ViewEncapsulation.None,
   host: { class: 'app-metadata-tab' }
 })
 export class MetadataTabComponent implements OnInit, OnDestroy {
   protected onDestroy$ = new Subject<boolean>();
-
   @Input()
   node: Node;
 
   displayAspect$: Observable<string>;
   canUpdateNode = false;
   editable = false;
+  editableTags = false;
+  editableCategories = false;
+  group: any = {
```

Comment:
can you avoid using any?

Reply:
I will update this once ADF upstream PR merges.

Reply:
ok please correct it after ADF upstream PR is merged and before you merge that ACA PR.

Reply:
Sure